sdk/devcenter/Azure.Developer.DevCenter/tests/DevBoxClientTests.cs

```diff
@@ -0,0 +1,417 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using System.Text.Json;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Developer.DevCenter.Tests
+{
+    [PlaybackOnly("It takes roughly 15-30 minutes to create a dev-box")]
```

Comment:
How do you plan to ensure that the recordings do not go stale? We do have live tests in other SDKs that can take time with resource creation. It is recommended to increase test timeout if that is what you are worried about. You can also choose to run tests at a different cadence which will still ensure you test it against the live service.

Reply:
We have a series of integration tests that run against the data plane API. So, if the behavior of an API version changes it would be captured by the integration tests first. So, we're not using the SDK live tests as integration tests for the Data Plane API.

I think it's reasonable to update the records with each new API version release.

Reply:
This is not a recommended approach. Playback mode is what is used in PRs and release pipelines so the cost of test runs would not be huge. You could chose to run  your live pipelines weekly or at a cadence that works better for you.  We do have a few other SDKs that can take a long time for resource creation.

I will not block on this however please consider creating a follow-up bug to fix this. Weekly pipelines can run for a longer period of time.